README.md

```diff
@@ -1,6 +1,9 @@
 # gds-pre-commit
 This repository is here to assist GDS users in setting up pre-commit hooks that can improve the quality and security of projects hosted on GitHub.
 
+## Secrets
+One of the main motivations for using pre-commit hooks is to prevent secrets being pushed to GitHub repositories. When we say secrets we mean things like private keys, API tokens, SSH keys, AWS keys or Slack keys. All of these 'secrets' are used to authenticate or authorise users to services we use or own and would be beneficial for an attacker to steal.
+
```

Comment:
should we add in here everything the pre-commit is likely to catch so that people don't think it'll catch everything?

Reply:
Maybe?? Or just add a note that this isn't a catch-all solution and people should follow best practices SSDLC and use secrets managers?